### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by liuxingbaoyu**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `Fixes #1, Fixes #2` 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,6 +1,3 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
```
Comment (liuxingbaoyu) [inline .husky/pre-commit:3]: https://github.com/typicode/husky/releases/tag/v9.0.1

**Review by JLHwung — APPROVED** [review]

**Comment by liuxingbaoyu** [comment]
The node 6 failure appears to be an acorn regression.

**Comment by JLHwung** [comment]
For the CI error on Node.js 6: it seems that we have to pin `acorn` in the `resolutions` package json field.

**Comment by liuxingbaoyu** [comment]
I tried
```
"acorn@npm:8.12.0": "npm:8.11.3",
```
and
```
"acorn@npm:^8.12.0": "npm:8.11.3",
```
Unfortunately neither worked, which is weird.

**Comment by JLHwung** [comment]
Indeed, I manually reverted the acorn upgrade and added a resolution field.

**Comment by liuxingbaoyu** [comment]
Amazing, thank you!

**Comment by JLHwung** [comment]
@liuxingbaoyu Now that the upstream issue has been fixed, can you bump acorn to `8.12.1`?

**Review by nicolo-ribaudo — APPROVED** [review]

**Comment by JLHwung** [comment]
Can you also remove the acorn resolution field?

**Pull request merged**